### DISCUSSION TITLE
how to use it with a bot

### DISCUSSION THREAD

**Discussion by 7xa5h** [Glitchii/embedbuilder#34] [Q&A] [answered]
how do i use the website with a discord bot

**Comment by Glitchii** [comment, marked as answer]
It's meant to be used in your website. Either use an iframe or clone the repository and the use it in your website. Cloning gives you more capabilities. You can alter the HTML/CSS to add a "send" button which when clicked should send the json data which is stored in `window.json` to Discord through a Discord bot. How you want to do that is up to you. Maybe have a request made to the backend with the json data or use websockets.
